src/core/tools/common/profiler_command.cpp

```diff
@@ -527,8 +634,8 @@ namespace dsn {
 
                 ss << "[";
                 for (int j = 0; j <= dsn_task_code_max(); j++)
-                    if (j != TASK_CODE_INVALID && task_spec::get(j)->pool_code == pool)
-                        ss <<"\""<< std::string(dsn_task_code_to_string(j)) << "\",";
+                if (j != TASK_CODE_INVALID && j != task_id && task_spec::get(j)->pool_code == pool && std::string(dsn_task_code_to_string(j)).find("_ACK") == std::string::npos)
```

Comment:
we should use taskspec::type instead of reply on the naming convention which is not reliable